src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -0,0 +1,533 @@
+<template>
+  <div>
+    <confirm-dialog ref="reassignCPDialog" attach="#addEditPersonContainer"/>
+    <v-expand-transition id="addEditPersonContainer">
+      <ul class="list add-person">
+        <li class="add-person-container">
+          <div class="meta-container">
+            <label class="add-person-header" v-if="isPerson">
+              <span v-if="activeIndex===-1">Add Person</span>
+              <span v-else>Edit Person</span>
+            </label>
+            <label class="add-person-header" v-if="isOrg">
+              <span v-if="activeIndex===-1">Add Corporation or Firm</span>
+              <span v-else>Edit Corporation or Firm</span>
+            </label>
+            <div class="meta-container__inner">
+              <v-form
+                ref="addPersonOrgForm"
+                class="appoint-form"
+                v-model="addPersonOrgFormValid"
+                v-on:submit.prevent="addPerson">
+                <label class="sub-header" v-if="isPerson">Person's Name</label>
+                <label class="sub-header" v-if="isOrg">Corporation or Firm Name</label>
+                <div class="form__row three-column" v-if="isPerson">
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="First Name"
+                    id="person__first-name"
+                    v-model="orgPerson.firstName"
+                    :rules="firstNameRules"
+                  />
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Middle Name"
+                    id="person__middle-name"
+                    v-model="orgPerson.middleName"
+                    :rules="middleNameRules"
+                  />
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Last Name"
+                    id="person__last-name"
+                    v-model="orgPerson.lastName"
+                    :rules="lastNameRules"
+                  />
+                </div>
+                <div v-if="isOrg" class="org-name-container">
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Full Legal Corporation or Firm Name"
+                    id="firm-name"
+                    v-model="orgPerson.orgName"
```

Comment:
Check if there's a maximum size for corp name.

Reply:
155 chars for Corp Name. Will apply!

Reply:
Applied the rule of 30 chars to the first, middle and last names 👍